css/_uswds-theme-custom-styles.scss

```diff
@@ -2906,11 +2840,6 @@ $theme-table-column-gap: 4;
       middle,
       $site-banner-background-color
     );
-    @include set-link-from-bg(
-      $site-banner-background-color,
-      $site-banner-link-color,
-      $context: $banner-context
-    );
 
     line-height: line-height($site-banner-font-family, 2);
     margin-bottom: units(0);
```

Comment:
**Suggestion:** Remove lines 2845-2855. Unable mark as a code suggestion due to unchanged lines.

```diff
-    margin-bottom: units(0);
-    margin-top: units(2px);
-    text-decoration: underline;
-    .usa-banner__header--expanded & {
-      display: none;
-    }
-    @include at-media("tablet") {
-      display: none;
-    }
```
Maintained by banner styles lines [172-182](https://github.com/uswds/uswds/blob/369cf3fdf9abf721a357e864a0f6adbf667ba862/packages/usa-banner/src/styles/_usa-banner.scss#L172-L182)